app/js/news/news.controller.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+  'use strict';
+  angular.module('cityInfo.news').controller('NewsController', NewsController);
+
+  NewsController.$inject = ['$http'];
+
+  function NewsController($http) {
+    var vm = this;
+    vm.news= [];
+
+    $http.get('https://newsapi.org/v1/articles?source=the-next-web&sortBy=latest&apiKey=4fd9c090e8c04c9f983076ed474461cd')
+    .success(function (response) {
```

Comment:
I don't really mind but in other places in this code base `.then` is used rather than `.success`.